tests/__init__.py

```diff
@@ -0,0 +1,3 @@
+import sys
+
+sys.path.append("../src/nervaluate")
```

Comment:
Do you assume here that the package won't be installed when running tests?

Reply:
exactly, I thought this would make more sense to run the tests directly against the current development version